Gems/Multiplayer/Code/Source/NetworkTime/NetworkTime.cpp

```diff
@@ -156,9 +156,10 @@ namespace Multiplayer
                             debugDisplay->DrawWireBox(rewoundAabb.GetMin(), rewoundAabb.GetMax());
                         }
 
-                        if (AZ::ShapeIntersection::Overlaps(rewoundAabb, rewindVolume)) // Validate the rewound aabb intersects our rewind volume
+                        if (AZ::ShapeIntersection::Overlaps(rewoundAabb, expandedVolume)) // Validate the rewound aabb intersects our rewind volume
```

Comment:
It would be better to test against the rewind volume? rewinding can be really expensive.. the expanded volume from earlier for the gather is simply because we don't rewind the entire visibility system just to run the gather operation.

Reply:
I think so too actually. I made this change before I realized what we're seeing with the shot's ray doesn't seem to line up with what we expect mathematically. I might be able to revert this change if that fix ends up getting it all to line up.